pkg/manager/member/tidbcluster_status_manager.go

```diff
@@ -125,6 +134,83 @@ func (m *TidbClusterStatusManager) syncDashboardMetricStorage(tc *v1alpha1.TidbC
 	return nil
 }
 
+// ref https://github.com/pingcap/tidb/blob/36b04d1aa01db722b3f07af759168c6b8da33801/domain/infosync/info.go#L72
```

Comment:
How about putting this ref above L34?

Reply:
keep here and add more comments.
I like to tell the reader how the two key is write but not just the definition of the prefix